<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-KK1SMTI" failures="1" tests="7" name="TestExecute" time="70.824" errors="0" timestamp="2024-11-25T22:17:23 IST">
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verifytest" time="12.71">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_003_LoginDataDrivenTest.verifytest(TC_003_LoginDataDrivenTest.java:41)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifytest -->
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verifytest" time="3.263"/>
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verifytest" time="2.355"/>
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verifytest" time="11.803"/>
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verifytest" time="2.758"/>
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verifytest" time="11.679"/>
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verifytest" time="11.788"/>
</testsuite> <!-- TestExecute -->
